django_irods/storage.py

```diff
@@ -99,15 +99,17 @@ def zipup(self, in_name, out_name):
         # SessionException will be raised from run() in icommands.py
         self.session.run("ibun", None, '-cDzip', '-f', out_name, in_name)
 
-    def unzip(self, zip_file_path):
+    def unzip(self, zip_file_path, unzipped_folder=None):
         """
         run iRODS ibun command to unzip files into a new folder
         :param zip_file_path: path of the zipped file to be unzipped
         :return: the folder files were unzipped to
         """
```

Comment:
@sblack-usu It'd be good to add the new optional parameter ```unzipped_folder``` into the docstring above.

Reply:
good catch, done